### DISCUSSION TITLE
Hello!

### DISCUSSION THREAD

**Discussion by Archi12351** [LiveHelperChat/livehelperchat#1998] [General]
Hello looking for an online converter to make an audio track from a video, who knows?

**Comment by PeopleInside** [comment]
Hi, this community is for discuss about Live Helper Chat issues.
Please don't abuse it :)

You can do a [Google search](https://www.google.com/search?&q=make+an+audio+track+from+a+video) about what you are looking for, the choice of with results follow, is yours.
